test/prow/patch_node_label_parallel.py

```diff
@@ -0,0 +1,230 @@
+                },
+                "placement": "Shared"
+            }
+        }
+        print(my_dataset)
+
+        my_alluxioruntime = {
+            "apiVersion": "data.fluid.io/v1alpha1",
+            "kind": "AlluxioRuntime",
+            "metadata": {"name": runtime},
+            "spec": {
+                "replicas": 1,
+                "podMetadata": {
+                    "labels": {
+                        "foo": "bar"
+                    }
+                },
+                "tieredstore": {
+                    "levels": [{
+                        "mediumtype": "MEM",
+                        "path": "/dev/shm",
+                        "quota": "2Gi",
+                        "high": "0.95",
+                        "low": "0.7"
+                    }]
+                }
+            }
+        }
+
+        api.create_namespaced_custom_object(
+            group="data.fluid.io",
+            version="v1alpha1",
+            namespace="default",
+            plural="datasets",
+            body=my_dataset,
+        )
+        print("Created dataset %s." % (runtime))
+
+        api.create_namespaced_custom_object(
+            group="data.fluid.io",
+            version="v1alpha1",
+            namespace="default",
+            plural="alluxioruntimes",
+            body=my_alluxioruntime
+        )
+        print("Created runtime %s" % (runtime))
+
+
+
+def checkDatasetBound(dataset):
+    api = client.CustomObjectsApi()
+
+    while True:
+        resource = api.get_namespaced_custom_object(
+            group="data.fluid.io",
+            version="v1alpha1",
+            name=dataset,
+            namespace="default",
+            plural="datasets"
+        )
+
+        print(resource)
+
+        if "status" in resource:
+            if "phase" in resource["status"]:
+                if resource["status"]["phase"] == "Bound":
+                    break
+        time.sleep(1)
+        print(resource)
+
+
+def cleanDatasetAndRuntime(*datasets):
+    custom_api = client.CustomObjectsApi()
+    for dataset in datasets:
+        custom_api.delete_namespaced_custom_object(
+            group="data.fluid.io",
+            version="v1alpha1",
+            name=dataset,
+            namespace="default",
+            plural="datasets"
+        )
+
+        runtimeDelete = False
+        while not runtimeDelete:
+            print("runtime %s still exists..." % (dataset))
+            try:
+                runtime = custom_api.get_namespaced_custom_object(
+                    group="data.fluid.io",
+                    version="v1alpha1",
+                    name=dataset,
+                    namespace="default",
+                    plural="alluxioruntimes"
+                )
+            except client.exceptions.ApiException as e:
+                if e.status == 404:
+                    runtimeDelete = True
+                    continue
+
+            time.sleep(1)
+
+
+def main():
+    config.load_kube_config()
```

Comment:
Please change it to `config.load_incluster_config()`

Reply:
ok